homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Me</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <main class="main">
+    <h1>About Me</h1>
```

Comment:
Hey, @AndriiMorozov88, please add styles (font-size, line-height) to `h1` and `p` according to [design](https://www.figma.com/file/rFE9mlgbxttnIECEpJSgyI/Masters-Academy-%7C-website-layout-for-course-task?type=design&node-id=47-994&mode=design&t=flO9pZPSK0ue8sCb-4)